pep/tests/integration/pods/components/modal-dialogs/user/about/component-test.ts

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | modal-dialogs/user/about', function(hooks) {
+    setupRenderingTest(hooks);
+
+    test('it renders', async function(assert) {
+        // Set any properties with this.set('myProperty', 'value');
+        // Handle any actions with this.set('myAction', function(val) { ... });
+
+        await render(hbs`{{modal-dialogs/user/about}}`);
+
+        assert.equal(this.element.textContent?.trim(), '');
+
+        // Template block usage:
+        await render(hbs`
+      {{#modal-dialogs/user/about}}
+        template block text
+      {{/modal-dialogs/user/about}}
+    `);
+
+        assert.equal(this.element.textContent?.trim(), 'template block text');
```

Comment:
this test is prob failing since the modal doesnt have a block form. I'm trying to see if we can always have completely green builds 100% for any code pushed to a release/ or master branches on this project.

I know not having tests turned on CI or blocking merges until they pass makes this harder to enforce, but I think its doable.

Reply:
Nothing failing except the random issues with GenericTaskProperty that we have talked about before